cli/src/commands/enrich/dependency_track.rs

```diff
@@ -0,0 +1,5 @@
+use clap::Parser;
+
+/// Args for use with the Dependency Track enrichment provider.
+#[derive(Clone, Debug, Parser)]
+pub struct DependencyTrackArgs {}
```

Comment:
Why is this here?  Does EPSS have anything to do with Dependency Track?  I'm serious about not bringing DT in to V2.  I will die on that hill.

Reply:
If Mike is still thinking it's a good idea to bring this to V2, we need to change his mind.  DT is a terrible fit for Harbor.

Reply:
@qtpeters  Regarding all the DT comments here:  I didn't introduce the DT stuff in this PR I just started splitting files up.  It's all already there on `main` from the initial `cli` PR which was merged before we ever talked about possibly not adding a DT provider. I had added it because at that time, we were definitely going to have one. I understand where you are coming from here, but that code change wasn't really intended to be a part of this PR.  It's showing up though, because as things are growing, I felt it made sense to started splitting the large files apart. 

I'll go ahead and remove this in this branch.  Thanks for the callout.